redeploy-citus

```diff
@@ -3,90 +3,151 @@
+kubectl exec $MASTER -- su - postgres -c "psql $DBNAME -c \"select run_command_on_workers(\\\$CITUS\\\$ DROP FUNCTION if exists kyrix_bbox_coords_identity \\\$CITUS\\\$);\""
+# master needs stable, workers need voltile
+kubectl exec $MASTER -- su - postgres -c "psql $DBNAME -c \"select run_command_on_workers(s) FROM ( select replace(pg_catalog.pg_get_functiondef('kyrix_bbox_coords_identity(int,int)'::regprocedure::oid)::text, 'STABLE', '') s)t;\""
+
+function tune_postgres() {
+    CONF=/var/lib/postgresql/data/postgresql.conf
+    kubectl exec $1 -- su - postgres -c "psql -c \"create extension if not exists plv8\" " # shutup start_proc warning
+    kubectl exec $1 -- sed -i "s@^[#]\?shared_buffers.*@shared_buffers = '$SHARED_BUFFERS_RAM'@" $CONF
+    kubectl exec $1 -- sed -i "s@^[#]\?work_mem.*@work_mem = '$WORK_MEM_RAM'@" $CONF
+    kubectl exec $1 -- sed -i "s@^[#]\?maintenance_work_mem.*@maintenance_work_mem = '1g'@" $CONF
+    kubectl exec $1 -- sed -i "s@^[#]\?max_worker_processes.*@max_worker_processes = '24'@" $CONF
+    kubectl exec $1 -- sed -i "s@^[#]\?max_parallel_workers.*@max_parallel_workers = '24'@" $CONF
+    kubectl exec $1 -- sed -i "s@^[#]\?max_parallel_workers_per_gather.*@max_parallel_workers_per_gather = '24'@" $CONF
+    kubectl exec $1 -- bash -c "echo \"plv8.start_proc = 'commonjs.plv8_startup'\" >> $CONF; echo \"citus.max_files_per_process = '5000'\" >> $CONF; echo \"citus.task_executor_type = 'task-tracker'\" >> $CONF; echo \"citus.max_connections = '500'\" >> $CONF"
+    kubectl exec $1 -- su - postgres -c "/usr/lib/postgresql/11/bin/pg_ctl -D /var/lib/postgresql/data reload -s"
+}
+
+echo "tuning postgres on each node, then reloading..."
+# set shared_buffers to 1/4 RAM - master/coordinator may have different RAM than workers
+MASTER_RAM=$(kubectl exec $MASTER free | grep Mem)
+SHARED_BUFFERS_RAM=$(echo $MASTER_RAM | perl -ane 'printf("%.2fg", $F[1]/1024/1024/4);')
+WORK_MEM_RAM=$(echo $MASTER_RAM| perl -ane 'printf("%.2fm", $F[1]/1024/100*3);')
+MAINTENANCE_MEM_RAM=$(echo $MASTER_RAM| perl -ane 'printf("%.2fm", $F[1]/1024/100*10);')
+tune_postgres $MASTER 
+
+WORKER_RAM=$(kubectl exec $FIRST_WORKER free | grep Mem)
+SHARED_BUFFERS_RAM=$(echo $WORKER_RAM | perl -ane 'printf("%.2fg", $F[1]/1024/1024/4);')
+WORK_MEM_RAM=$(echo $WORKER_RAM| perl -ane 'printf("%.2fm", $F[1]/1024/100*3);')
+MAINTENANCE_MEM_RAM=$(echo $WORKER_RAM| perl -ane 'printf("%.2fm", $F[1]/1024/100*10);')
+for w in $CITUS_ALL_NODES; do tune_postgres $w; done > /dev/null
+
+echo "testing citus..."
+./test-citus
+echo "testing plv8..."
+./test-plv8
+#./test-postgis
+
+echo "installing d3 on master and every worker..."
+for w in $CITUS_ALL_NODES; do kubectl exec $w -- su - postgres -c "./install-d3.sh $DBNAME"; done > /dev/null
```

Comment:
`install-d3.sh` is only called once here. Does this mean single node won't have d3 pushdowns?

Reply:
good catch. Technically, pushdown could/should be done separately. In practice,
I was hoping to get this in then backport pushdown to single node so this PR doesn't hang out for another month+.